api-server/api-server-common/src/storage/storage_api/mod.rs

```diff
@@ -64,6 +68,47 @@ pub enum ApiServerStorageError {
     TimestampToHigh(BlockTimestamp),
 }
 
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Debug)]
+pub enum CoinOrTokenStatistic {
+    TotalSupply,
```

Comment:
It seems like it should be `CirculatingSupply` to conform with chainstate terminology where TotalSupply is the limit for minting and Circulating is current amount minted/unminted. But I'm confused why this arm is used for coins in transfer outputs.

Reply:
the transfer output is probably the arm from block reward, those are from genesis.